cordova-react/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",         
+        "jsx": "react",
+        "sourceMap": true,
```

Comment:
i thought that Cordova uses `inlineSoruces` and `inlineSourceMap` to get debugging working across browsers, emulators, etc..